sdk/program/src/bpf_loader_upgradeable.rs

```diff
@@ -119,6 +119,11 @@ impl UpgradeableLoaderState {
     }
 }
 
+/// Returns the program data address for a program ID
+pub fn get_program_data_address(program_address: &Pubkey) -> (Pubkey, u8) {
```

Comment:
Just to point out: in bpf loader upgradeable, it's typically called `programdata` without the underscore, but loader v4 calls it `program_data`. The second formulation is probably better, so I'm fine with this

Reply:
Yeah I saw that, and I'm not going to lie `programdata` was driving me bananas.

Reply:
haha totally fair!